### DISCUSSION TITLE
scala-cli from within Emacs

### DISCUSSION THREAD

**Discussion by ag91** [VirtusLab/scala-cli#2501] [Show and tell]
Hey, thanks for maintaining scala-cli!

A little while ago I made a little mode to use scala-cli from within Emacs: https://github.com/ag91/scala-cli-repl
I really like to do literate programming and I wrote the mode to integrate with [Org Mode](https://orgmode.org/).

My question is: is there a space in the docs to share about these editor integrations? 
I wonder if other people would find it useful too (I shared it on Reddit, but I hope you could suggest a better place).

Thanks again!

**Comment by Gedochao** [comment]
Awesome! What do you think about adding it as a [guide](https://scala-cli.virtuslab.org/docs/guides/intro) or a [cookbook](https://scala-cli.virtuslab.org/docs/cookbooks/intro)?

**Reply by Gedochao** [reply]
For example, we already have cookbooks for IDE integrations. Check the [VS Code](https://scala-cli.virtuslab.org/docs/cookbooks/vscode) & [IntelliJ](https://scala-cli.virtuslab.org/docs/cookbooks/intellij) entries.

**Reply by ag91** [reply]
oh that is great! I will open a PR as soon as I write down something. Thanks!
